packages/mux-player/src/video-api.ts

```diff
@@ -128,13 +128,7 @@ class VideoApiElement extends HTMLElement {
   }
 
   get hasPlayed() {
-    const mc = this.shadowRoot?.querySelector('media-controller');
```

Comment:
with the theme change, this no longer works, so, we should grab it via `this.mediaController` instead.